sympy/matrices/expressions/matmul.py

```diff
@@ -13,7 +13,7 @@
 from sympy.matrices.matrices import MatrixBase
 
 
-class MatMul(MatrixExpr, Mul):
+class MatMul(MatrixExpr):
```

Comment:
I thought subclassing makes sense.

Reply:
See the issue I opened for why this causes problems.